camelot/view/controls/editors/textlineeditor.py

```diff
@@ -112,6 +114,7 @@ def set_field_attributes(self, **kwargs):
                                          kwargs.get('background_color'))
             if completer:
                 text_input.setCompleter(completer)
+            validator = self.get_validator(self.validator_type, validator_state)
```

Comment:
en dan kan je 'set_validator_type' generiek implementeren als findChildByName('validator), if is not None, validator.setState(bla bla bla)